dub.json

```diff
@@ -18,7 +18,7 @@
             "excludedSourceFiles": ["src/main.d"],
             "dependencies": {
                 "unit-threaded": "~>0.7.0",
```

Comment:
While at it, shouldn't `unit-threaded` be at **0.7.43**, as in `dub.selections.json`?

Reply:
No, because of [this dub bug](https://github.com/dlang/dub/issues/1217). I dare not even update uni-threaded to 0.8 or it'd break everything.

Reply:
No, because of [this dub bug](https://github.com/dlang/dub/issues/1217). I dare not even bump up unit-threaded to 0.8 or else everything would break.